content/docs/iac/concepts/vs/cloud-template-transpilers/aws-cdk/_index.md

```diff
@@ -34,6 +34,41 @@ Choosing the right infrastructure as code tool is important, and we want you to
+- __Testing__. Pulumi supports fast, in-memory unit tests that mock external calls, enabling developers to validate infrastructure without deploying it. CDK testing is limited to assertions against synthesized CloudFormation templates, which slows down the feedback loop.
+
+### Security
+
+Pulumi natively integrates security features with infrastructure provisioning.
+
+- __Secrets Management__. Pulumi has built-in secrets management that encrypts secrets both in transit and at rest. It also integrates seamlessly with external secrets managers like AWS Secrets Manager. CDK has no built-in support for managing or handling encrypted secrets.
+- __Policy as Code__. Pulumi enables your organization to enforce security and compliance policies during deployment using Pulumi CrossGuard's policy as code. This feature is tightly integrated into Pulumi's workflows, whereas CDK offers only limited support for such policies. __[NEED TO VALIDATE]__
+
+### Savings
+
+Pulumi can deliver cost savings over CDK due to its multi-cloud support and efficiency gains in the inner development loop.
+
+- __Multi-Cloud Support__. Pulumi supports over 160 cloud providers, allowing your organization to optimize costs by choosing the most cost-effective provider for your needs. CDK is limited to AWS, which may lock you into higher cost AWS services. Using CDK will also necessitate employing another IaC system to manage non-AWS resources which will increase training time for new engineers and operational complexity.
+- __Operational Efficiency__. Pulumi’s faster deployments reduce engineering time spent on debugging and iteration cycles. This translates into lower development time costs compared to the more time-intensive development workflows required by CDK.
+- __Interoperability__. Pulumi can integrate with existing resources managed by other IaC tools, eliminating the need for costly migrations or extensive code refactoring. CDK lacks interoperability with other IaC tools, which requires full migration to CDK code in order to interact with existing resources.
+
+### Scale
+
+Pulumi is better equipped to handle large-scale deployments across multi-cloud environments.
+
+- __Provider Ecosystem__. Pulumi supports multi-cloud environments, Kubernetes, SaaS providers, and on-premises infrastructure. This flexibility enables organizations to scale across different platforms without being constrained by a single provider like AWS in the case of CDK.
+- __Resource Limits__. Unlike CloudFormation (and by extension CDK), which imposes a 500-resource limit per template, Pulumi imposes no hard limits on the number of resources in a stack, making it better suited for complex deployments.
+- __Workflow Orchestration__. With its Automation API, Pulumi allows programmatic management of infrastructure at scale. This feature enables dynamic orchestration workflows that are not possible with CDK's CLI-based execution model.
```

Comment:
Automation API is the last bullet?!  It's one of the biggest differentiators! Let's not bury the lede.